app/models/application_record.rb

```diff
@@ -7,4 +7,10 @@ def text_format(text)
     text = 'Keine Angabe' if text.blank?
     simple_format(text)
   end
+
+  def self.document_content_types
```

Comment:
```ruby
def self.ext_mimes(*extensions)
  extensions.flat_map do |ext|
      MIME::Types.select { |type| type.extensions.include?(ext.to_s) }
  end.map(&:to_s)
end

# .... 

validates :attachment, attached: true,
                                     content_type: ext_mimes(:xls, :xlsx, :doc, :docx, :odt, :ods, :pdf)


validates :image, content_type: ext_mimes(:jpg, :gif, :png, :tif, :webp)
```

I had that in mind. But no big deal. Never mind

Reply:
oh, yes, this is much better